corehq/apps/app_manager/templates/app_manager/partials/module_option_button.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
+
+<button type="button" class="new-module-option" data-type="{{ module_type }}">
+  <div class="panel panel-default">
+    <div class="panel-heading">{% trans heading %}</div>
+    <div class="panel-body">
+      <i class="{{ icon_class }}"></i>
+      <p>{% blocktrans %}{{ description }}{% endblocktrans %}</p>
```

Comment:
the blocktrans here will not work as you expect. Instead it will present " {{ description }}" as the translation string (with `description` as a variable. Same with `{% trans heading %}` above. that's why i suggest template extending